frontend/src/pages/Setup/SetupRouter/AlertsSetup.tsx

```diff
@@ -299,30 +331,24 @@ const AlertPicker = function ({
 		fetchPolicy: 'network-only',
 	})
 
-	const [createSessionAlert] = useCreateSessionAlertMutation({
-		refetchQueries: [namedOperations.Query.GetAlertsPagePayload],
-	})
+	const [createSessionAlert] = useCreateSessionAlertMutation()
 
-	const [createErrorAlert] = useCreateErrorAlertMutation({
-		refetchQueries: [namedOperations.Query.GetAlertsPagePayload],
-	})
+	const [createErrorAlert] = useCreateErrorAlertMutation()
 
-	const [createLogAlert] = useCreateLogAlertMutation({
-		refetchQueries: [namedOperations.Query.GetAlertsPagePayload],
-	})
+	const [createLogAlert] = useCreateLogAlertMutation()
 
-	const [upsertSlackChannel] = useUpsertSlackChannelMutation({
-		refetchQueries: [namedOperations.Query.GetAlertsPagePayload],
-	})
-	const [upsertDiscordChannel] = useUpsertDiscordChannelMutation({
-		refetchQueries: [namedOperations.Query.GetAlertsPagePayload],
-	})
+	const [upsertSlackChannel] = useUpsertSlackChannelMutation()
+	const [upsertDiscordChannel] = useUpsertDiscordChannelMutation()
 
 	const hasDefaultSessionAlert = data?.new_session_alerts.find(
 		(a) => a?.default,
 	)
 	const hasDefaultErrorAlert = data?.error_alerts.find((a) => a?.default)
 	const hasDefaultLogAlert = data?.log_alerts.find((a) => a?.default)
+	const hasChanges =
+		(alertsSelected.indexOf('Session') === -1) != !hasDefaultSessionAlert ||
```

Comment:
This is perfectly fine, but you can probably use `alertsSelected.includes('Session')` instead of `indexOf`.

Reply:
great point, i always forget about `includes` and `find`